run_test.sh

```diff
@@ -233,9 +233,15 @@ while [[ "$#" -gt 0 ]]; do
         shift 1
         ;;
     --explore)
-        echo --explore "### Running explore page integration tests"
-        run_integration_test explore_test.py
-        shift 1
+        if [[ -n "$2" && "$2" != -* ]]; then
```

Comment:
what does `-*` mean? :)

Reply:
this is to check the argument is not like "-p" as an option (from llm..)